bugsnag-android-core/src/main/java/com/bugsnag/android/Stacktrace.kt

```diff
@@ -27,31 +27,52 @@ internal class Stacktrace : JsonStream.Streamable {
             }
             return null
         }
-    }
 
-    val trace: List<Stackframe>
-    val logger: Logger
+        fun stacktraceFromJavaTrace(
+            stacktrace: Array<StackTraceElement>,
+            projectPackages: Collection<String>,
+            logger: Logger): Stacktrace {
+            val frames = limitTraceLength(stacktrace.mapNotNull { serializeStackframe(it, projectPackages, logger) })
```

Comment:
The NDK will also need to limit the number of stackframes to 200 - this seems fine as long as the requirement isn't forgotten when implementing a similar method to achieve this via the JNI call.

Reply:
Yes, it will use the same `BUGSNAG_FRAMES_MAX ` as https://github.com/bugsnag/bugsnag-android/blob/master/bugsnag-plugin-android-ndk/src/main/jni/event.h#L166